### ISSUE TITLE
[Implementation-Change] Observing 404 for forwarding contruct and forwarding-construct-port APIs.

### ISSUE THREAD

**Issue by Ziabatcha** [openBackhaul/MultiDomainInventoryProxy#26]
Depends on #33

Tested with v1.0.1_test

Observing 404 for forwarding contruct and forwarding-construc-port APIs.
Lookslike the "=" is not added for fc and fc-port, hence API fails.

For Both APIs we are able to see the response In MWDI.

```

​/core-model-1-4:network-control-domain=cache​/control-construct={mountName}​/forwarding-domain={uuid}​/forwarding-construct{uuid1}

/core-model-1-4:network-control-domain=cache/control-construct={mountName}/forwarding-domain={uuid}/forwarding-construct{uuid1}/fc-port{localId}

```

```

"http://xxx:xxx/core-model-1-4:network-control-domain=cache/control-construct=xxx/forwarding-domain=FD-VLAN-1234-1/forwarding-constructFC-1234-1-3392" -H "accept: application/json" -H "Authorization: Basic YWRtaW46QWRtaW4xMjM="

{
  "code": 404,
  "message": "not found"
}


g-constructFC-1234-1-3392","response-code":404} failed with response status: 400
[04:57:03.651] ERROR (1): error during /core-model-1-4:network-control-domain=cache/control-construct=513250004/forwarding-domain=FD-VLAN-1234-1/forwarding-constructFC-1234-1-3392
    err: {
      "message": "Request failed with status code 404",
      "stack":

   }
loggingErrorHandler - caught error, returning response with status code "404" and body {"code":404,"message":"not found","errors":[{"path":"/","message":"not found"}]}
HEAD / 404 11.617 ms - 122
recordServiceRequest - record service request with body {"application-name":"MultiDomainInventoryProxy","release-number":"1.0.0","operation-name":"/","response-code":404} failed with response status: 400
POST /v1/update-operation-key 204 15.74
```

```
/core-model-1-4:network-control-domain=cache/control-construct={mountName}/forwarding-domain={uuid}/forwarding-construct{uuid1}/fc-port{localId}

curl -X GET "http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250004/forwarding-domain=FD-VLAN-1234-1/forwarding-constructFC-1234-1-3392/fc-portFCPort-1234-1-3392" -H "accept: application/json" -H "Authorization: Basic YWRtaW46QWRtaW4xMjM="


{
  "code": 404,
  "message": "not found"
}

loggingErrorHandler - caught error, returning response with status code "404" and body {"code":404,"message":"not found","errors":[{"path":"/","message":"not found"}]}
HEAD / 404 12.283 ms - 122
recordServiceRequest - record service request with body {"application-name":"MultiDomainInventoryProxy","release-number":"1.0.0","operation-name":"/","response-code":404} failed with response status: 400

```

**Comment by MFMentopolis** [comment]
Looking in the github history of the MicroWaveDeviceInventory, I see recent changes in the MWDI API:

The errors we are faced are due to a mismatch between the modified MWDI and the current MDIP implementation:
The way forward is to adjust the MDIP specification to match that of the MWDI and then adapt the implementation accordingly.

**Comment by kmohr-soprasteria** [comment]
Created new issue #33 to update the specification. Once this issue is resolved, the implementation shall be changed and further tracked with this issue.

**Comment by kmohr-soprasteria** [comment]
Updated loadFile

**Comment by paulBMentopolis** [comment]
Fixed in tag "v1.0.0_impl-fix-4"

**Comment by Ziabatcha** [comment]
Latest tag : [v1.0.0_impl-fix-4] test observation:
Issue can be re-tested only when MWDI/MDIP specification is in sync with parameter as 'fc' instead of 'forwarding-construct'.

http://xx:xx/core-model-1-4:network-control-domain=cache/control-construct=xx/forwarding-domain=xx/forwarding-construct=xx

**Comment by kmohr-soprasteria** [comment]
changed forwarding-construct to fc with #44

**Comment by paulBMentopolis** [comment]
Fixed in tag "v1.0.0_impl-fix-5"

Tests on Lab:
/core-model-1-4:network-control-domain=cache/control-construct={mountName}/forwarding-domain={uuid}/fc={uuid1}

with input data:
mountName: 513250008
uuid: VLAN-FD
uuid1: VLAN-201
fields: layer-protocol-name

result: 
{
  "core-model-1-4:fc": [
    {
      "layer-protocol-name": "vlan-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_VLAN_LAYER"
    }
  ]
}

/core-model-1-4:network-control-domain=cache/control-construct={mountName}/forwarding-domain={uuid}/fc={uuid1}/fc-port={localId}

with input data:
mountName: 513250010
uuid: VFD-CB
uuid1: VLANFC-500
localId: 10

result:
{
  "core-model-1-4:fc-port": [
    {
      "local-id": "10",
      "lifecycle-state": "core-model-1-4:LIFECYCLE_STATE_INSTALLED",
      "logical-termination-point": [
        "VI-10"
      ],
      "operational-state": "core-model-1-4:OPERATIONAL_STATE_ENABLED",
      "fc-port-direction": "core-model-1-4:PORT_DIRECTION_BIDIRECTIONAL",
      "selection-priority": "0",
      "administrative-state": "core-model-1-4:ADMINISTRATIVE_STATE_UNLOCKED",
      "is-internal-port": false,
      "administrative-control": "core-model-1-4:ADMINISTRATIVE_CONTROL_QUIESCENT",
      "is-protection-lock-out": false
    }
  ]
}

**Comment by Ziabatcha** [comment]
Tested OKAY with v1.0.0_impl-fix-5 and issue is not observed.

```
Logs:
{core-model-1-4:forwarding-domain: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250011/forwarding-domain=VFD-CB/fc=VLANFC-500
200
88 ms
 
{core-model-1-4:fc: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250004/forwarding-domain=FD-VLAN-1234-1/fc=FC-1234-1-500
200
130 ms
 
{core-model-1-4:fc: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250006/forwarding-domain=VLAN-FD/fc=VLAN-806/fc-port=9
200
102 ms
 
[Uploading FC-FCport_MDIP.txt…]()

{core-model-1-4:fc-port: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250006/forwarding-domain=VLAN-FD
200
104 ms
 
{core-model-1-4:forwarding-domain: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250006/forwarding-domain=VLAN-FD/fc=VLAN-806
200
91 ms
 
{core-model-1-4:fc: [1]}
 
GET http://172.28.127.10:3013/core-model-1-4:network-control-domain=cache/control-construct=513250006/forwarding-domain=VLAN-FD/fc=VLAN-806/fc-port=9
200
79 ms

```
